### ISSUE TITLE
End python 2 support

### ISSUE THREAD

**Issue by MaxLeiter** [devsnd/cherrymusic#676]
Leads to annoying CI failures such as: https://travis-ci.org/devsnd/cherrymusic/jobs/255910887

honestly, just switch to python 3

**Comment by devsnd** [comment]
I agree ([see here #280](https://github.com/devsnd/cherrymusic/issues/280)).

But before we decide something like that using a gut feeling, maybe we should check if the most popular distros (especially for raspberry pi and the like) already switched to python 3 by default, etc. This might be a more reasonable thing to act upon before we break everybodies CM setup!

**Comment by MaxLeiter** [comment]
Ah, I actually was mostly in favor of this due to the failing tests, but now I see those are due to `pypy` -- any chance you can remove/fix that test, @devsnd?
